docs/topics/gradle/gradle-backend-tutorial.md

```diff
@@ -0,0 +1,112 @@
+[//]: # (title: Get started with Gradle and Kotlin/JVM)
+
+This tutorial demonstrates how to use IntelliJ IDEA and Gradle for creating a console application.
+
+To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
+
+## Create a project
+
+1. In IntelliJ IDEA, select **File** | **New** | **Project**.
+2. In the panel on the left, select **New Project**.
+3. Name the new project and change its location, if necessary.
+
+   > Select the **Create Git repository** checkbox to place the new project under version control. You will be able to do
+   > it later at any time.
+   >
+   {type="tip"}
+
+4. From the **Language** list, select **Kotlin**.
+
+   ![Create a console application](jvm-new-gradle-project.png){width=700}
+
+5. Select the **Gradle** build system.
+6. From the **JDK list**, select the [JDK](https://www.oracle.com/java/technologies/downloads/) that you want to use in
+   your project.
+    * If the JDK is installed on your computer, but not defined in the IDE, select **Add JDK** and specify the path to the
+      JDK home directory.
+    * If you don't have the necessary JDK on your computer, select **Download JDK**.
+
+7. From the **Gradle DSL** list, select **Kotlin**.
+8. Select the **Add sample code** checkbox to create a file with a sample `"Hello World!"` application.
+9. Click **Create**.
+
+You have successfully created a project with Gradle.
+
+## Explore the build script
+
+Open the `build.gradle.kts` file. This is the Gradle Kotlin build script, which contains a list of the dependencies required for the application.
+
+Here is the full script with the explanation of all parts and dependencies:
+
+```kotlin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile // For `KotlinCompile` task below
+
+plugins {
+    kotlin("jvm") version "%kotlinVersion%" 1️⃣ // The version of Kotlin to use
```

Comment:
When you are limited of space (like the product UI, documentation comments in code, other similar places), you can focus on how to make your text shorter but useful.

For example, you can omit articles and change wording to avoid a lot of prepositions and conjunctions.
`// Kotlin version to use` does the same job for me as `The version of Kotlin to use`.